corehq/form_processor/generic.py

```diff
@@ -201,6 +204,10 @@ def reverse_indices(self):
         from corehq.form_processor.interfaces.case import CaseInterface
         return CaseInterface.get_reverse_indices(self.domain, self.id)
 
+    @property
+    def is_deleted(self):
+        return self.doc_type.endswith(DELETED_SUFFIX)
```

Comment:
OK for now but should probably be a property that gets set when converting from XFormInstance e.g. https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/couchforms/models.py#L451-451